app/pages/api/realms/one.ts

```diff
@@ -172,9 +184,22 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
             division=$7,
             branch=$8,
             updated_at=now()
+            second_technical_contact_email=$11,
```

Comment:
can you double-check this indexes are the correct ones?

Reply:
Likely not, they are $15 and $16 at line #119